src/index.tsx

```diff
@@ -158,73 +167,128 @@ export class App extends React.Component<{}, {
         var data = this.state.results;
 
-        if (!data) 
-        {
+        if (!data) {
             return <div>No search results.</div>
         }
-        if (data["RecId"].length > 1) 
-        {
-            return <div>Must reduce to a single record. You can use RecId criteria to specify an exact record. </div>
+
+        if (data["RecId"].length > 1 && !this.state.selectedRecId) {
+            return (
+                <div>
+                    Must reduce to a single record or select one from the table above.
+                    You can use RecId criteria to specify an exact record.
+                </div>
+            )
         }
-        return <div><p>Fill in answers for: {data["RecId"][0]}</p>
-            <AllQuestions 
-                onSubmit={this.onSubmitAnswers}
-                columns={_trcGlobal._info.Columns}
-            ></AllQuestions>
+
+        let index = 0;
+
+        if (this.state.selectedRecId) {
+            index = data["RecId"].findIndex((x) => x === this.state.selectedRecId);
+        }
+
+        return (
+            <div>
+                <p>Fill in answers for: {data["RecId"][index]}</p>
+                <AllQuestions
+                    onSubmit={this.onSubmitAnswers}
+                    columns={_trcGlobal._info.Columns}
+                />
             </div>
+        )
 
     }
 
     private renderBody1() {
 
-        return <div>
-            <h2>Search</h2>
-            <p>This searches for specific individuals within the sheet and will also let you update answers for them. Use <PluginLink id={"Filter"}></PluginLink> to get counts. </p>
-            <div>Enter Search criteria: (Blanks match everything)</div>
-            <div>{_trcGlobal._info.CountRecords} total records. </div>
-            <FieldInputs Names={["RecId", "First", "Last", "Address", "City", "Zip", "PrecinctName"]} onSubmit={this.onSearch}></FieldInputs>
-
-            <h3>Search Results</h3>
-            {this.state.results && <SimpleTable data={this.state.results} ></SimpleTable>}
-            {this.state.totalFound  &&  <p>Found {this.state.totalFound} total result(s).</p>}
-            {(this.state.totalFound  && this.state.totalFound > this._topN) &&  <p>Showing first {this._topN} results. Narrow the search further.</p>}
-
-            <h3>Edit answers</h3>
-            {this.renderQuestions()}
-        </div>
+        return (
+            <PluginShell
+                description={
+                    <>
+                        <p>
+                            This searches for specific individuals within the sheet and
+                            will also let you update answers for them.
+                        </p>
+                        <p>
+                            Use <PluginLink id={"Filter"}></PluginLink> to get counts.
+                        </p>
+                    </>
+                }
+                title="Search"
+            >
+                <Panel>
+                    <Grid>
```

Comment:
What are the basic layout rules for what to expect from <grid> / <copy>?

Reply:
At the moment <Grid> renders `children` exclusively in a 2 columns layout with a `1rem` gap between each of them.

When the time arises, the plan is to make <Grid> customizable via props in terms of how many columns it should render (we will keep the default at 2). Another customization option could be the gap between columns.